### PR TITLE
Prepare pre-release r1.1

### LINKED ISSUE #71: Why "The API can be called by an application server or other 3rd party server"
**Problem description**

> The API can be called by an application server or other 3rd party server to establish the identity of the device currently being used by the mobile subscription.

https://github.com/camaraproject/DeviceIdentifier/blob/main/code/API_definitions/CAMARA%20Mobile%20Device%20Identifier%20API.yaml#L39

**Expected action**
All Camara APIs can be called from any device, be it server, smartphone, laptop.

Not sure, why this sentence is here.

If somebody thinks that calling the API from a server is a security "requirement" then this should be documented.
If it is common practice to send the OIDC code or the access token to a server, and then call a Camara API from there, then that adds an additional place where the code or access token can be stolen or leaked. There should be a risk assessment.

For this API I suggest removing that sentence.

**Additional context**
My recommendation: 
- Native Apps should use [PKCE](https://www.rfc-editor.org/rfc/rfc7636)
- Camara should use sender-constraint access tokens. See e.g. [DPoP](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-security-topics#name-sender-constrained-access-t).

Sending the authentication code or access tokens from the OAuth2 client, that requested them, to a different server changes the OAuth2 security considerations.

### PR DESCRIPTION
**Pull request by eric-murray**
#### What type of PR is this?
* subproject management

#### What this PR does / why we need it:
Preparation of release r1.1 for device-identifier v0.2.0-alpha.1

- [x] Update CHANGELOG.med
- [x] Update README.md
- [x] Set versions and url in device-identifier.yaml
- [x] Update API Readiness Checklists

#### Which issue(s) this PR fixes:
Fixes #71 

#### Special notes for reviewers:
None

#### Changelog input

```
 release-note
 - Preparation of release r1.1 for device-identifier v0.2.0-alpha.1
```

#### Additional documentation 
N/A

### CODE REVIEW

**Review by hdamker — DISMISSED** [review]
Please add @camaraproject/release-management_maintainers when ready for review.

Regarding the release number: I also learned only recently that the release numbers are not tied to the meta-release, but start always for an API repository with r1.1, c.f. https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/API_Release_Guidelines.md:

> The release numbers shall follow the guidelines described below.
> * Release numbers start with x=1 and y=1: r1.1.
> * y is incremented by 1 at each subsequent alpha, release-candidate and public release, and for a maintenance release, e.g. rx.y+1.
> * After a meta-release of an API through release rx.y, the next release number for this API is rx+1.1 (y resets to 1).
> * In case of maintenance of a release rx.y, the new public release shall be rx.y+1.

Given that the release number should be r1.1 - @tanjadegroot correct?

**Comment by eric-murray** [comment]
All references to r2.1 changed to r1.1

File: code/API_definitions/device-identifier.yaml
Comment (hdamker) [inline code/API_definitions/device-identifier.yaml:1]: Lines 124/125 (contact field) should not be included.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+## Release Notes
+
+This public release contains the definition and documentation of
+* device-identifier v0.2.0-alpha.1
+
+The API definition(s) are based on
+* Commonalities v0.4.0
+* Identity and Consent Management v0.2.1
+
+## quality-on-demand v0.2.0-alpha.1
```
Comment (hdamker) [inline CHANGELOG.md:19]:
```suggestion
## device-identifier v0.2.0-alpha.1
```

**Review by hdamker — CHANGES_REQUESTED** [review]
One minor, but important c&p mistake to be corrected

**Review by hdamker — APPROVED** [review]

**Comment by hdamker** [comment]
I've created a release tracker within the wiki. Please check: https://lf-camaraproject.atlassian.net/wiki/x/cABkAQ (especially if you plan to go with 0.2.0 into the Spring25 meta-release, or only with the next initial release).

**Comment by eric-murray** [comment]
> One minor, but important c&p mistake to be corrected

Thanks @hdamker. Thought I'd caught them all.

Re-adding @camaraproject/release-management_maintainers should anybody else want to review.

**Comment by hdamker** [comment]
@eric-murray There were no further comments within today's Release Management call. So you are good to go as soon as @Kevsy has approved.

Just the release tracker needs to be replaced as we today in RM changed the template again a little bit. I can take that as soon as the template is updated (as the last step of the release review).

**Review by Kevsy — APPROVED** [review]

**Comment by hdamker** [comment]
@eric-murray @Kevsy PR is ready to get merged and release created.

And: I have recreated the release tracker based on the latest template updates. Can you please check and decide if 0.2.0 is the target version for Spring25 or do you plan the public release earlier and a later version for Spring25? Depending on that please update the first line of the release tracker and the label on the page: https://lf-camaraproject.atlassian.net/wiki/x/boNkAQ

**Comment by tanjadegroot** [comment]
> Please add @camaraproject/release-management_maintainers when ready for review.
> 
> Regarding the release number: I also learned only recently that the release numbers are not tied to the meta-release, but start always for an API repository with r1.1, c.f. https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/API_Release_Guidelines.md:
> 
> > The release numbers shall follow the guidelines described below.
> > 
> > * Release numbers start with x=1 and y=1: r1.1.
> > * y is incremented by 1 at each subsequent alpha, release-candidate and public release, and for a maintenance release, e.g. rx.y+1.
> > * After a meta-release of an API through release rx.y, the next release number for this API is rx+1.1 (y resets to 1).
> > * In case of maintenance of a release rx.y, the new public release shall be rx.y+1.
> 
> Given that the release number should be r1.1 - @tanjadegroot correct?

Yes, correct

File: code/API_definitions/device-identifier.yaml
Comment (tanjadegroot) [inline code/API_definitions/device-identifier.yaml:1]: line 102: the the --> that the

Comment (tanjadegroot) [inline code/API_definitions/device-identifier.yaml:1]: line 124: remove "contact" section (it is to be added by the implementor)

File: documentation/API_documentation/API Readiness Checklist device-identifier.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+| 12 | Previous public release was certified        |   O   |         O         |    O    |    M   |  N   |                                                                        |
+
+To fill the checklist:
+- in the line above the table, replace the api-name, api-version and the rx.y by their actual values for the current API version and release.
+- in the Status column, put "Y" (yes) if the release asset is available or fulfilled in the current release, a "N" (no) or a "tbd". Example use of "tbd" is in case an alpha or release-candidate API version does not yet provide all mandatory assets for the release.
+- in the Comments column, provide the link to the asset once available, and any other relevant comments.
+
+Note: the checklists of a public API version and of its preceding release-candidate API version can be the same.
+
+The documentation for the content of the checklist is here: [API Readiness Checklist](https://wiki.camaraproject.org/display/CAM/API+Release+Process#APIReleaseProcess-APIreadinesschecklist).
```
Comment (tanjadegroot) [inline documentation/API_documentation/API Readiness Checklist device-identifier.md:27]: change link to new wiki: https://lf-camaraproject.atlassian.net/wiki/spaces/CAM/pages/14559630/API+Release+Process#API-readiness-checklist

File: code/API_definitions/device-identifier.yaml
[the remark this answers is not in the document]
Comment (hdamker) [reply code/API_definitions/device-identifier.yaml:1]: > Line 32 should be a relative link (no reference to "main")

That wouldn't work if the YAML is used outside the GitHub context, e.g. in swagger editor. Instead the main has to be replaced with the release. But I wouldn't do that for alpha releases, here it don't matter if the picture get updated until the release candidate.

**Review by Kevsy — APPROVED** [review]

**Pull request merged**